### DISCUSSION TITLE
Welcome to eslint-plugin-compat Discussions!

### DISCUSSION THREAD

**Discussion by amilajack** [amilajack/eslint-plugin-compat#430] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eightHundreds** [comment]
![image](https://user-images.githubusercontent.com/18695431/146150298-7b6d009f-c4e5-4c27-ac39-6379605b96d7.png)

What is this IDE?

**Reply by amilajack** [reply]
It's Atom
